hack/kubearchive-install.sh

```diff
@@ -17,13 +17,8 @@ PODS=$(kubectl -n kubearchive get pods | grep -E -v "NAME|No resources|apiserver
 RELEASE_VERSION=$(cat VERSION)
```

Comment:
The code that use the RELEASE_VERSION is not needed now, right? Then, I think we can delete these lines